tests/e2e/docs/vanilla_cluster_setup.md

```diff
@@ -80,6 +81,12 @@ list of datastore URLs where you want to deploy file share volumes. Retrieve thi
     3.cat ~/.ssh/id_rsa.pub | ssh root@<vc-ip-address> 'cat >> .ssh/authorized_keys'
     4.ssh root@<vc-ip-address> "chmod 700 .ssh; chmod 640 .ssh/authorized_keys"
 
+#### Setting SSH keys for ESX with your local machine to run VCP to CSI migration static provisioning tests
```

Comment:
We do not need additional setup instructions if esx host IP can be fetched dynamically

Reply:
this is for password less login to the esx host to run commands on the esx host just like we are doing for VC